.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Docker
```

Comment:
@Sammers21 What about `Publish Docker Image` ?

Reply:
I would change it to `Docker Release`, just for consistency with file name